src/components/Calculator.js

```diff
@@ -1,64 +1,55 @@
-import React from 'react';
+import { useState } from 'react';
 import calculate from '../logic/calculate';
+import Button from './Button';
 
 // eslint-disable-next-line react/prefer-stateless-function
-class Calculator extends React.Component {
```

Comment:
- [ ] Great job so far, it is better to solve the error than disable the linter, kindly fix the errors, if any.